src/main/java/desktop/pages/AccountPage.java

```diff
@@ -2,71 +2,55 @@
 
 import abstractclasses.page.AbstractPage;
 import desktop.fragments.NavigationBarFragment;
+import driver.SingletonDriver;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
-public class AccountPage extends AbstractPage {
+import static constants.Constants.SIGN_IN_PAGE_URL;
+
 
+public class AccountPage extends AbstractPage {
+    @FindBy(xpath = "//h1[contains(@class,'a-spacing-top-small')]")
+    public WebElement titleText;
     @FindBy(xpath = "//*[@id='ap_email']")
     WebElement emailField;
-
-    @FindBy(xpath = "//div[contains(@class,'a-input-text-group')]//input[@type='password']")
+    @FindBy(xpath = "//input[@type='password']")
     WebElement passwordField;
-
-    @FindBy(xpath = "//form[@name='signIn']//input[@id='signInSubmit']")
+    @FindBy(xpath = "//input[@id='signInSubmit']")
     WebElement submitButton;
-
-    @FindBy(xpath = "//h1[contains(@class,'a-spacing-top-small')]")
-    public WebElement titleText;
+    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
 
     public AccountPage(WebDriver driver) {
         super(driver);
-    }
-
-    NavigationBarFragment navigationBar = new NavigationBarFragment(driver);
-
-    private static String accountPageUrl = "https://www.bookdepository.com/account/login/to/account";
-    private static String successfulLoginPageURL = "https://www.bookdepository.com/ap/signin";
-
-
-    public void openPage() {
-        getDriver().get(accountPageUrl);
+        SingletonDriver.getInstance();
     }
 
     public boolean checkURL() {
```

Comment:
it is better to make such methods void and do the assertion inside - currently it just returns equal value and doesn't check anything

Reply:
fixed, but not sure it's done in the correct way

Reply:
yes, it is correct, but please swap argument (expected is the first one) and add an error message:
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/90613721/164033413-1396b05a-9e7f-4175-9068-dded74f322bb.png">

Reply:
now it's correct, resolved this thread